src/components/Aboutus.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import "./Aboutus.css";
+import CardDeck from 'react-bootstrap/CardDeck';
+import Card from 'react-bootstrap/Card';
+import image from "../images/why-us.png";
+import mission from "../images/mission.png";
```

Comment:
All images should be in the public folder.so put all images in public and change the path.

Reply:
![Screenshot (83)](https://user-images.githubusercontent.com/62712155/108062038-39559680-707f-11eb-9946-ec09d91d01a6.png)
I am getting an error. Please have a look.

Reply:
> ![Screenshot (83)](https://user-images.githubusercontent.com/62712155/108062038-39559680-707f-11eb-9946-ec09d91d01a6.png)
> I am getting an error. Please have a look.

No need to maintain a ```./../public/xyz.png```  path you can only write the direct name of images like ```src="images/xyz.png"```.
Try with this.

Reply:
I did this, now it's not showing an error. Please have a look.